src/lib/components/ui/TextField/__tests__/TextField.test.jsx

```diff
@@ -48,7 +48,20 @@ describe('rendering', () => {
     ...inFormLayoutPropTest,
     [
       { inputSize: 3 },
-      (rootElement) => expect(within(rootElement).getByRole('textbox')).toHaveAttribute('size', '3'),
+      (rootElement) => {
+        expect(within(rootElement).getByRole('textbox')).toHaveAttribute('size', '3');
+        expect(rootElement).toHaveStyle('--rui-custom-input-size: 3');
+      },
+    ],
+    [
+      {
+        inputSize: 3,
+        type: 'number',
+      },
+      (rootElement) => {
+        expect(within(rootElement).getByLabelText('label')).not.toHaveAttribute('size', '3');
```

Comment:
The `size` attribute shouldn't be there at all, no matter its value.

Reply:
good catch!